scripts/build_palettes.ts

```diff
@@ -43,5 +44,26 @@ Promise.all(
       join(ROOT, `sip/${fname}.palette`),
       generateSip(fname, colors),
     );
+
+    const clrJson = join(ROOT, `clr/${fname}.json`);
+    await Deno.writeTextFile(
+      clrJson,
+      generateClrJson(fname, colors),
+    );
+    const cmd = new Deno.Command("swift", {
+      args: [
+        join(import.meta.dirname!, "./builders/palettes/json-to-clr.swift"),
+        clrJson,
```

Comment:
Specifically titled the files this way because this is what the name of each palette is when you view them in a supported color picker app. Helps to have the Catppuccin part, maybe we add this to the others?

Reply:
![CleanShot 2024-09-06 at 07 24 32](https://github.com/user-attachments/assets/de6828a7-f03e-4827-bfa8-a7c9d8121567)

You can see what I mean visually here- currently all the other palette files are just the capitalized identifier. I've tested and can confirm that the color picker apps do support the accent, and having the word "Catppuccin" in there is good for clarity (as we do with theme files for other ports).

Reply:
Happy to accept another PR updating the code to prefix these files with "Catppuccin" :+1:

Reply:
Cool, will do.